content/it/devops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: DevOps
+status: Completed
+category: concept
+---
+
+## Cos'è
+DevOps è una metodologia in cui i team sono responsabili dell'intero processo dallo sviluppo delle applicazioni fino alle attività di installazione, configurazione e manutenzione in produzione, da cui DevOps. DevOps vá oltre l'implementazione di una serie di tecnologie e richiede un cambiamento completo nella cultura e nei processi. DevOps richiede gruppi di ingegneri che lavorano su piccoli componenti (invece di un'intera feature), diminuendo i passaggi di mano - una fonte comune di errori. 
+
+## Quale problema affronta
+Tradizionalmente, nelle organizzazioni complesse con [tightly-coupled](/tightly_coupled_architectures/) [monolithic apps](/monolithic_apps/), il lavoro era generalmente frammentato tra più team. Questo portava a numerosi passaggi di mano e a lunghi tempi di consegna. Ogni volta che un componente o un aggiornamento era pronto, veniva messo in una coda per il team successivo. Poiché gli individui lavoravano solo su una piccola parte del progetto, questo approccio portava ad una mancanza di proprietà sull'intero progetto. Il loro obiettivo era quello di passare il lavoro al gruppo successivo e non di consegnare la giusta funzionalità al cliente - un evidente e chiaro disallineamento delle priorità.
+
+Quando il codice arrivava finalmente in produzione, passava attraverso così tanti sviluppatori ed in attesa in così tante code che era difficile rintracciare l'origine del problema se il codice non funzionava. DevOps capovolge questo approccio.
+
+## In che modo aiuta
+Avere un team che è responsabile dell'intero ciclo di vita di un'applicazione porta a minimizzare i passaggi di mano, a ridurre il rischio quando si installa in produzione, a migliorare la qualità del codice perché i team sono anche responsabili di come il codice si comporta in produzione e ad aumentare la soddisfazione dei dipendenti grazie alla maggiore autonomia e proprietà.
```

Comment:
anche in questo caso _ownership_ lo lascerei invariato, oppure lo tradurre con titolarità di progetto

Reply:
certo, per ora meglio ownership, titolarità non credo che sia proprio adeguato, magari cerca un altro sinonimo